src/openrct2/ride/Vehicle.cpp

```diff
@@ -7173,10 +7173,10 @@ void Vehicle::UpdateSpinningCar()
  *
  *  rct2: 0x006734B2
  */
-static void steam_particle_create(int16_t x, int16_t y, int16_t z)
+static void steam_particle_create(const CoordsXYZ& coordinates)
```

Comment:
```suggestion
static void steam_particle_create(const CoordsXYZ& coords)
```

We use coords often in the code, so I guess this shortened one would be nice.

Reply:
Done as suggested!